adagrams/game.py

```diff
@@ -1,11 +1,115 @@
+import random
+import string
+
+
 def draw_letters():
-    pass
+    LETTER_POOL = ETTER_POOL = {
+    'A': 9, 
+    'B': 2, 
+    'C': 2, 
+    'D': 4, 
+    'E': 12, 
+    'F': 2, 
+    'G': 3, 
+    'H': 2, 
+    'I': 9, 
+    'J': 1, 
+    'K': 1, 
+    'L': 4, 
+    'M': 2, 
+    'N': 6, 
+    'O': 8, 
+    'P': 2, 
+    'Q': 1, 
+    'R': 6, 
+    'S': 4, 
+    'T': 6, 
+    'U': 4, 
+    'V': 2, 
+    'W': 2, 
+    'X': 1, 
+    'Y': 2, 
+    'Z': 1
+}
+    hand = []
+    count_frequency = {}
+    for letter in LETTER_POOL:
+        count_frequency[letter] = 0
+        
+    i = 0
+    while i < 10:
```

Comment:
Instead of using a variable and keeping count for yourself, you can do `while len(hand) < 10`